sedaro/src/sedaro/branches/common.py

```diff
@@ -49,6 +49,10 @@ def __getattr__(self, key: str) -> any:
         if not self.is_rel_field(key):
             return val
 
+        # If relationship is undefined, return None
+        if val is None:
+            return None
+
```

Comment:
Quick thought on this. What would you think of moving this conditional under `if side_type == ONE_SIDE:`?

It should only ever be `None` when it's a "OneSide", but in case there were ever an incorrect `None` in a `ManySide` or `DataSide`, that could help make sure not to mask it.

Reply:
Sure, I can change that

Reply:
On second thought, I'm going to leave it.  I agree that we wouldn't want to mask it in certain places but the client should be fault tolerant when used on corrupt models where possible. Otherwise it makes fixing the corrupt models impossible.  My change improves fault tolerance in the instances where a ManySide or DataSide come through as None.